src/LiveComponent/assets/dist/Component/index.d.ts

```diff
@@ -13,8 +13,8 @@ export type ComponentHooks = {
     'response:error': (backendResponse: BackendResponse, controls: {
         displayError: boolean;
     }) => MaybePromise;
-    'loading.state.started': (element: HTMLElement, request: BackendRequest) => MaybePromise;
-    'loading.state.finished': (element: HTMLElement) => MaybePromise;
+    'loading.state:started': (element: HTMLElement, request: BackendRequest) => MaybePromise;
+    'loading.state:finished': (element: HTMLElement) => MaybePromise;
```

Comment:
Took me some time to understand this ... as you have some CS changes mixed in here :) 

Could you -if you have time- open a PR just for this "bug fix" ? 

I know it's only TS and not _really_ a bug, but people will probably see these values in their IDE... and i'd really like them to see the good ones instead 😅  

And also to ensure we merge the fix asap .... because this one will depend on the path we choose to take :)

Reply:
Will do! I just added the typing to the hooks to not have to remember the event names and having a better DX, but I was wondering why my event wasn't firing... Only to find out there was a typo in the typing :sweat_smile: